Description: avoid breaking change by accepting the current upstream
 value with underscore, and the value specified in the spec with a dash
Upstream: https://github.com/spdx/tools-java/pull/211

diff --git a/resources/spdx-schema-v2.3.json b/resources/spdx-schema-v2.3.json
index 403d202..36c955a 100644
--- a/resources/spdx-schema-v2.3.json
+++ b/resources/spdx-schema-v2.3.json
@@ -413,7 +413,7 @@
           "primaryPackagePurpose" : {
             "description" : "This field provides information about the primary purpose of the identified package. Package Purpose is intrinsic to how the package is being used rather than the content of the package.",
             "type" : "string",
-            "enum" : [ "OTHER", "INSTALL", "ARCHIVE", "FIRMWARE", "APPLICATION", "FRAMEWORK", "LIBRARY", "CONTAINER", "SOURCE", "DEVICE", "OPERATING_SYSTEM", "FILE" ]
+            "enum" : [ "OTHER", "INSTALL", "ARCHIVE", "FIRMWARE", "APPLICATION", "FRAMEWORK", "LIBRARY", "CONTAINER", "SOURCE", "DEVICE", "OPERATING_SYSTEM", "OPERATING-SYSTEM", "FILE" ]
           },
           "releaseDate" : {
             "description" : "This field provides a place for recording the date the package was released.",
